components/autocomplete/index.js

```diff
@@ -136,6 +136,7 @@ export class Autocomplete extends Component {
 		this.search = this.search.bind( this );
 		this.handleKeyDown = this.handleKeyDown.bind( this );
 		this.getWordRect = this.getWordRect.bind( this );
+		this.debouncedLoadOptions = debounce( this.loadOptions, 250 );
```

Comment:
Any time we debounce, we must also be certain to cancel any scheduled invocations if the component unmounts in the time between, i.e.:

```js
componentWillUnmount() {
	this.debouncedLoadOptions.cancel();
}
```

Reply:
Good point - addressed in https://github.com/WordPress/gutenberg/pull/4406/commits/3202fb79cc805753895a2e60c36a9723970095c8.